addons/mail/models/mail_identity.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, tools
+
+
+class MailNotification(models.Model):
+    _name = 'mail.identity'
+    _description = 'Identity'
+    _order = 'id desc'
+
+    # description
+    name = fields.Char(string='Name')
+    # contact
+    email = fields.Char(
+        string='Email', compute='_compute_email',
+        readonly=False, store=True)
+    email_normalized = fields.Char(
+        string='Normalized Email', compute="_compute_email_normalized",
+        compute_sudo=True, store=True)
+    phone = fields.Char(
```

Comment:
maybe add phone formating

Reply:
Indeed, it we keep a phone number this will probably be done with an override in phone_validation.